experimental/dds/tree2/src/test/snapshots/files/insert-and-delete-tree-0-final.json

```diff
@@ -19,22 +19,28 @@
                       "change": {
                         "changes": [
                           {
-                            "fieldKey": "rootFieldKey",
-                            "fieldKind": "Sequence",
-                            "change": [
-                              {
-                                "count": 1,
-                                "effect": {
-                                  "delete": {
-                                    "id": 1
-                                  }
+                            "data": {
+                              "changes": [
+                                {
```

Comment:
do we actually need to add 3 levels of nesting here? That seems like a bit much.

Reply:
I'm fine with working on that as a followup, but I think we might want to track it, and I think we might have some design patterns that lead to more nesting overhead than we really want in our persisted data

Reply:
Will address before merging.

Reply:
Done!